packages/wrangler/src/__tests__/type-generation.test.ts

```diff
@@ -482,6 +486,7 @@ describe("generateTypes()", () => {
 		export {};
```

Comment:
Should I update this to ES Module or there's a reason we have Service Worker syntax in this test?

Reply:
This is intended, to ensure we properly generate the global interface